### ISSUE TITLE
Determine way to allow for clearing Model / Discarding Data

### ISSUE THREAD

**Issue by vanschroeder** [Webfreshener/JSD#35]
Once a model is set, old values might linger around if not directly overwritten
Should provide a means to clear or completely overwrite the model on update even if certain elements are not overwritten by the new user data

```
// user set this:
model = {
  valueA: 1,
  valueB: 2,
  valueC: 3
}

// then sets this:
model = {
  valueA: 10,
  valueB: 12,
}

// model is then {valueA: 10, valueB: 12, valueC: 3}
```

it should be noted that in some cases this might be desired, this should be a user setting on the JSD document itself, perhaps even per Schema Object or Element

**Comment by vanschroeder** [comment]
recategorized as bug, since this is a pretty basic and expected feature

**Comment by vanschroeder** [comment]
fixed with commit af36838b2de334d3797eb5d28a3358d1d8828b53
